.circleci/config.yml

```diff
@@ -21,6 +21,43 @@ jobs:
             dep check
 
 
+  lint:
+    docker:
+      - image: circleci/golang:1.14
+    environment:
+      GOPATH: /home/circleci/.go_workspace
+      GOLANGCI_VERSION: v1.20.0
+    working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
+    steps:
+      # Workaround for custom env vars not available in cache keys
+      # https://discuss.circleci.com/t/cannot-use-circle-yml-environment-variables-in-cache-keys/10994/8
+      - run: echo $GOLANGCI_VERSION > /tmp/.golangci.version
+      - restore_cache:
+          name: Restore golangci-lint cache
+          key: golangci-lint-{{ arch }}-{{ checksum "/tmp/.golangci.version" }}-{{ .Environment.CACHE_VERSION }}
+          paths:
+            - /home/circleci/.go_workspace/bin/golangci-lint
+      - run:
+          name: Install golangci-lint
+          command: |
+            export PATH="$GOPATH/bin:$PATH"
+            command -v golangci-lint && exit
+            GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@$GOLANGCI_VERSION
+      - save_cache:
+          name: Save golangci-lint cache
+          key: golangci-lint-{{ arch }}-{{ checksum "/tmp/.golangci.version" }}-{{ .Environment.CACHE_VERSION }}
```

Comment:
I added `CACHE_VERSION` to the project's environment variables as a cache busting mechanism, since there doesn't appear to be any other way to clear the cache... :facepalm:

Reply:
Given the fact that we want to get out of circleci, I wouldn't have spent any time on cache at all :smile: if you make the linting happen in parallel with the testing, nobody is going to notice a minute more downloading the linter, the test will still be much slower :sweat_smile:

Reply:
More like 10s actually :D Every bit counts! :D

But yeah, it would've saved me from reading their support forum. :see_no_evil:

Reply:
yeah I also think that caching is not important at this point ... If it works - great, if it doesn't drop it :D.

Reply:
It works now, with the minor workaround for env vars, so I'll leave it in.